interface/patient_file/report/custom_report.php

```diff
@@ -308,7 +308,36 @@ function postToGet($arin) {
     //
     if (stristr($key,"include_")) {
 
-        if ($val == "demographics") {
+        if($val == "recurring_days"){
+
+            $recurrences = fetchRecurrences($pid);
+            if($recurrences[0] == false){ //if there are no recurrent appointments:
+                echo "<div>";
+                echo "<span>" . xlt('None') . "</span>";
+                echo "</div>";
+                echo "<br>";
+            }
+            else {
+                foreach ($recurrences as $row) {
+                    //checks if there are recurrences and if they are current (git didn't end yet)
+                    if ($row == false || !recurrence_is_current($row['pc_endDate']))
+                        continue;
+                    echo "<div>";
+                    echo "<span>" . xlt('Appointment Category') . ': ' . text($row['pc_title']) . "</span>";
```

Comment:
translate via a special command for appointment categories:
``````
echo "<span>" . xlt('Appointment Category') . ': ' . text(xl_appt_category($row['pc_title'])) . "</span>";